packages/app/README.md

```diff
@@ -0,0 +1,17 @@
+# NFT Passports App
+
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app). The app is accessible at app.nftpassports.xyz.
```

Comment:
Took out some of the create-next-app readme boilerplate and added a sentence/title specific to each project